### PR TITLE
Us31

### LINKED ISSUE #31: Merchant Items Grouped by Status
As a merchant,
When I visit my merchant items index page
Then I see two sections, one for "Enabled Items" and one for "Disabled Items"
And I see that each Item is listed in the appropriate section

### PR DESCRIPTION
**Pull request by gregoryjflaherty**
## Description of Changes

* Added tests, instance method in merchant model, as well as changed the view to fit user story. This user story shows an enabled/disabled item section in the view

## Concerns (tags recommended)

* had issues with the later tests in spec file having access to the data in the before(:each). Tried for the longest time to figure out why. Had to add a second before(:each). Will try and refactor this and remove later. 

closes #31

## What gif describes how you feel about this PR?

alice-rabbithole

### CODE REVIEW

File: app/views/merchant_items/index.html.erb [hunk 1]
```diff
@@ -1,12 +1,19 @@
 <h1><%= @merchant.name %>'s Items</h1>
-<% @merchant.items.each do |item| %>
-  <section id="item<%=item.id%>">
-      <h4><%= link_to "#{item.name}", "/merchants/#{@merchant.id}/items/#{item.id}" %> </h4>
-      <p> Enabled: <%= item.enabled %></p>
-    <% if item.enabled == true%>
-      <%= button_to "Disable #{item.name} Item", merchant_item_path(@merchant.id, item.id, enabled: true), {method: 'patch'} %>
```
Comment (glsorensen) [inline app/views/merchant_items/index.html.erb:7]: Nice, i will use this way of calling a specific path in a link going forward

File: app/views/merchant_items/index.html.erb [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+<div class="enabled_items">
+  <h3>Enabled Items</h3>
+  <% @merchant.enabled_disabled_items(true).each do |item| %>
+    <h4><%= link_to "#{item.name}", "/merchants/#{@merchant.id}/items/#{item.id}" %></h4>
+    <p> Enabled: <%= item.enabled %></p>
+    <%= button_to "Disable #{item.name} Item", merchant_item_path(@merchant.id, item.id, enabled: true), {method: 'patch'} %>
+  <% end %>
+</div>
+
+<div class="disabled_items">
```
Comment (glsorensen) [inline app/views/merchant_items/index.html.erb:12]: Nice use of divs to be able to test better

File: spec/features/merchant_items/index_spec.rb [hunk 3]
```diff
@@ -11,45 +11,97 @@
...
+        expect(page).to_not have_button("Disable #{@af_one.name} Item")
         end
       end
     end
   end
+
+  describe 'User Story 31' do
+    describe 'has sections for enabled and disabled items and lists those items in each section' do
+      before(:each) do
+        @nike = Merchant.create!(name: "Nike")
```
Comment (glsorensen) [inline spec/features/merchant_items/index_spec.rb:53]: Nice use of the divs in your tests

**Review by glsorensen — APPROVED** [review]

**Review by jgrazulis — APPROVED** [review]
Looks good, love the robust testing. I did notice our model and feature coverage is below 100% so we should definitely take a look at the model coverage to see what we're missing.

**Pull request merged**